src/applications/dependents/686c-674/config/chapters/674/addStudentsArrayPages.js

```diff
@@ -505,9 +505,9 @@ export const studentTermDatesPage = {
           'ui:description': TermDateHint,
         },
         expectedStudentStartDate: {
-          ...currentOrPastDateUI(
+          'ui:title':
             'When did the student start or expect to start their course?',
-          ),
+          'ui:webComponentField': VaMemorableDateField,
           'ui:required': () => true,
```

Comment:
We also need to include a `ui:errorMessages` `required` message

<img width="477" alt="Screenshot 2025-06-11 at 12 31 12 PM" src="https://github.com/user-attachments/assets/ce23b24f-eddd-4484-b472-d749dc6e4b03" />